libqtile/backend/wayland/inputs.py

```diff
@@ -279,7 +281,7 @@ def _on_key(self, _listener: Listener, event: KeyboardKeyEvent) -> None:
                 xkb_keysym,
             )
             keysyms = [xkb_keysym[0][i] for i in range(nsyms)]
-            mods = self.keyboard.modifier
+            mods = reduce(or_, [k.keyboard.modifier for k in self.core.keyboards])
```

Comment:
Can you write this as `any()`?

Reply:
I am not very sure how the later program treats `mods`, but I don't think `any()` works; [`wlroots.wlr_types.keyboard.Keyboard.modifier`](https://github.com/flacjacket/pywlroots/blob/ebafb5aeb65821134980b9e939d406bfeb222f55/wlroots/wlr_types/keyboard.py#L148) returns [`KeyboardModifier`](https://github.com/flacjacket/pywlroots/blob/ebafb5aeb65821134980b9e939d406bfeb222f55/wlroots/wlr_types/keyboard.py#L25), which is a [bitwise flag](https://gitlab.freedesktop.org/wlroots/wlroots/-/blob/3b3ed21e61b871abdad83f0ee769effdb7d43e56/include/wlr/types/wlr_keyboard.h#L30), not a boolean. `mods` must retain its bitwise representation to distinguish which modifier key is pressed and that's why I use bitwise-or `operator.or_`, not logical-or (`or`) which is compatible with `any()`.